katello/helper.rb

```diff
@@ -13,9 +13,7 @@ def error_message
     end
 
     def foreman_rpm_installed?
-      # using backticks here so there is no output
-      `rpm -q foreman`
-      $?.success?
+      system("rpm -q foreman >/dev/null")
```

Comment:
Either way works fine for me as long as there is no output.  Do you mind adding a space after then `>` pipe?

Reply:
If we are going to update to anything across the scripts, using http://ruby-doc.org/stdlib-1.9.3/libdoc/open3/rdoc/Open3.html#method-c-popen3 would be ideal. It starts a new external process that can be monitored. I'm not saying this should be done in this PR, but would be worth looking into more.

Reply:
We definitely do not want to update across the scripts at this time. system() returns a true/false/nil value, and run_cmd returns output. Updating with the space now :+1: